ansible/roles/backup/tasks/main.yml

```diff
@@ -0,0 +1,149 @@
+---
```

Comment:
Can we give this role a more specific name?
It looks really specific, so maybe `gitcoin-ceramic-backup`?

Reply:
I'll just take it out for now until we need it.